src/makeRobotAccountant.js

```diff
@@ -6,7 +6,20 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let countOfOperations = 0;
+  const getSum = (x) => {
+    countOfOperations++;
+
+    return (y) => {
+      if (countOfOperations >= 3 && countOfOperations % 2 === 0) {
+        return 'Bzzz... Error!';
+      }
+
+      return x + y;
+    };
+  };
+
+  return getSum;
 }
 
```

Comment:
i think this solution is more understandable, but you can do it as you want. You don't need getSum and return it

```
function makeRobotAccountant() {
  let countOfOperations = 0;

  return x => {
    return y => {
      countOfOperations++;

      if (countOfOperations >= 3 && countOfOperations % 2 === 0) {
        return 'Bzzz... Error!';
      }

      return x + y;
    };
  };
}
```

Reply:
Yeah, you are right. I'll make it more convenient. Thank you for the review.